src/templates/_partials/_cta.twig

```diff
@@ -0,0 +1,12 @@
+<div class="cta-bg h-[280px] lg:h-[371px] flex items-center justify-center p-[15px] rounded-md lg:p-0 lg:rounded-none">
+  <div class="flex flex-col lg:flex-row justify-between py-9 px-[30px] gap-4 lg:gap-0 bg-white w-full max-w-screen-narrow items-center">
+    <h4 class="text-center h4 lg:text-start">
+      Take better control of your biological and contaminant removal process.
+    </h4>
+    <button class="px-8 py-4 rounded-full bg-secondary">
+      <span class="text-lg font-semibold leading-[21px] text-white">
+        Schedule a Call
+      </span>
+    </button>
```

Comment:
Use created btn-outline class in the button element as the base and add tweaks after, and use an anchor tag instead of button.